setup.cfg

```diff
@@ -1,93 +0,0 @@
-[tool:pytest]
-addopts = --doctest-modules
-; use "--cov-report term-missing --cov ." to also run coverage.
-; remember to clear numba cache and disable Numba JIT for accurate coverage.
-; use "make numba-clean" and "export NUMBA_DISABLE_JIT=1".
-norecursedirs = doc
-
-[metadata]
-name = skan
-version = 0.11.1
-url = https://jni.github.io/skan
-download_url = https://github.com/jni/skan
-license = BSD 3-Clause
-license_file = LICENSE
-description = Skeleton analysis in Python
-long_description = file: README.md
-long_description_content_type = text/markdown
-author = Juan Nunez-Iglesias
-author_email = juan.nunez-iglesias@monash.edu
-classifiers =
-    Development Status :: 3 - Alpha
-    Environment :: Console
-    Intended Audience :: Developers
-    Intended Audience :: Education
-    Intended Audience :: Science/Research
-    License :: OSI Approved :: BSD License
-    Programming Language :: Python
-    Programming Language :: Python :: 3 :: Only
-    Programming Language :: Python :: 3.8
-    Programming Language :: Python :: 3.9
-    Programming Language :: Python :: 3.10
-    Topic :: Scientific/Engineering
-    Operating System :: Microsoft :: Windows
-    Operating System :: POSIX
-    Operating System :: Unix
-    Operating System :: MacOS
-    Framework :: napari
-
-
-[options]
-zip_safe = False
-setup_requires =
-    wheel
-package_dir =
-    =src
-packages = find:
-python_requires = >=3.8
-include_package_data = True
-install_requires =
-    imageio>=2.10.1
-    matplotlib>=3.4
-    networkx>=2.7
-    numba>=0.53
-    numpy>=1.21
-    pandas>=2.0.2
-    openpyxl>=2.6
-    scikit-image>=0.17.1
-    scipy>=1.7
-    toolz>=0.10.0
-    tqdm>=4.57.0
-
-[options.packages.find]
-where = src
```

Comment:
I think this is probably the issue @ns-rse: `where = src` whereas in pyproject.toml you've set `where = [.]`?

Reply:
Good spot, thank you, I thought `where = [.]` would make `find` traverse the directory structure looking for `__init__.py` to identify where packages were to be included from and had explicitly said to include the `src`.

Corrected and tested in a fresh environment and it looks like it installs ok and `napari` starts and has `skan` listed as a plugin.

I've also tried building the wheel (`python -m build`) and then installing the resultant wheel in a fresh Python 3.10 environment and it too installs and `napari` runs and has the `skan` plugin available.